scripts/generate_kubernetes_code.py

```diff
@@ -376,6 +378,13 @@ def autogenerate_triggers(f: TextIO):
     for resource in KUBERNETES_RESOURCES:
         f.write(f"# {resource} Triggers\n")
         for trigger_name, operation_type in sorted(TRIGGER_TYPES.items()):
+            if f"{resource}{get_trigger_class_name(trigger_name)}" in [
```

Comment:
can you explain this?

Reply:
As was discussed previously, only the event types listed here should have `change_filters` set to `DEFAULT_CHANGE_FILTERS`, the reason being change tracking for the other events makes no sense, right?

I think this change was previously made to the code under `src/robusta/integrations/kubernetes/autogenerated/`, and all the changes in `generate_kubernetes_code.py` in this PR are introduced just to make sure the autogenerated code after running the script is exactly the same as the code in the repo.

Reply:
As discussed, my reasoning here was actually wrong and all of these classes should have `change_filters` default set to `None`, which has been fixed.